.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+# 2.1 does not yet support local run
+# unless with workaround. For simplicity just use it.
+# https://github.com/CircleCI-Public/circleci-cli/issues/79
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
```

Comment:
Shouldn't this be `checksum package-lock.json`?

Reply:
DONE. You are right, Most CircleCI config uses package.json instead of package-lock.json. It turns out that tradition seems to start before NPM supports a `package-lock.json`.